demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -874,6 +875,8 @@ def _collect_pack_items(pack_id, id_set):
         pack_items['classifiers'] = PackDependencies._search_for_pack_items(pack_id, id_set['Classifiers'])
         pack_items['mappers'] = PackDependencies._search_for_pack_items(pack_id, id_set['Mappers'])
         pack_items['widgets'] = PackDependencies._search_for_pack_items(pack_id, id_set['Widgets'])
+        pack_items['dashboards'] = PackDependencies._search_for_pack_items(pack_id, id_set['Dashboards'])
```

Comment:
are all content entities now covered?

Reply:
I'm not sure what other content entities there are, so I don't know if i'd say that, but there are no failures currently on this step.

And the dashboards and reports are not covered, they are only collected as items in the pack now, but the dependencies are not calculated from them. I opened another issue for that here https://github.com/demisto/etc/issues/30118

Reply:
so should we add a TODO here for that ?